### ISSUE TITLE
BUG: Certification does not contain a full Standard object

### ISSUE THREAD

**Issue by jcscottiii** [opencontrol/compliance-masonry#224]
This is a bug from the beginning actually.

The certification struct looks like this:
```go
// Certification struct is a collection of specific standards and controls
// Schema info: https://github.com/opencontrol/schemas#certifications
type Certification struct {
	Key       string              `yaml:"name" json:"name"`
	Standards map[string]v1.Standard `yaml:"standards" json:"standards"`
}
```

And the Standard (with it's enclosed Control) looks like this

```go
// Control struct stores data on a specific security requirement
// Schema info: https://github.com/opencontrol/schemas#standards-documentation
type Control struct {
	Family string `yaml:"family" json:"family"`
	Name   string `yaml:"name" json:"name"`
}

// Standard struct is a collection of security requirements
// Schema info: https://github.com/opencontrol/schemas#standards-documentation
type Standard struct {
	Name     string             `yaml:"name" json:"name"`
	Controls map[string]Control `yaml:",inline"`
}
```

so theoretically, the certification.yaml should look like

```yml

name: FedRAMP-low
standards:
  NIST-800-53:
   AC-1:
     family: AC
     name: Access Control Policy and Procedures
```
but it actually looks like 

```yml

name: FedRAMP-low
standards:

  NIST-800-53:
    AC-1: {}
    AC-14: {}
```

it purposefully does not have all that information such as `family` and `name` because it's already documented in the standard.yaml

So the Certification struct should look like (might be wrong btw)

```go
type Certification struct {
	Key       string              `yaml:"name" json:"name"`
	Standards map[string][]string `yaml:"standards" json:"standards"`
}
```

Just need to stop using `v1.Standard`

In addition, the `common.Certification` that is yet to be created should not have a method called `GetStandard`. Instead, it should have a method such as `GetControlKeysFor(standardKey string)` and will return the []string of the map

This is probably going to be taken care of in #215

**Comment by afeld** [comment]
Related to https://github.com/opencontrol/schemas/issues/53, yeah?

**Comment by jcscottiii** [comment]
Yeah, having that would help to catch it a lot earlier!

It's something that was there since the beginning when it was ported to Go so having the docs up to date would've help when writing it.

**Comment by afeld** [comment]
Fixed by #215.
